src/btree/bt_cursor.c

```diff
@@ -1663,28 +1697,29 @@ __wt_btcur_equals(WT_CURSOR_BTREE *a_arg, WT_CURSOR_BTREE *b_arg, int *equalp)
 	}
 	return (0);
 }
-
 /*
  * __cursor_truncate --
- *	Discard a cursor range from row-store or variable-width column-store
- * tree.
+ * Discard a cursor range from row-store or variable-width
```

Comment:
There seems to be a lot of editing noise in this file that isn't related to the commits that you're backporting. It'd be helpful to compare the original commits with yours to double-check that you're only including the desired changes.

Reply:
done.